### PR TITLE
Update GitLab to version 8.14.1

### PR DESCRIPTION
**Pull request by twk3**
This is an update to the latest GitLab version.

In addition to new features, this version includes several security fixes that have gone in since our last DC/OS release.

The release blog post for 8.14 is here: https://about.gitlab.com/2016/11/22/gitlab-8-14-released/

### CODE REVIEW

**Comment by ryadav88** [comment]
@twk3 Can you list the new feature and security fixes in a comment here? 
We've started these monthly blog posts to let users know what's changed in different packages: https://dcos.io/blog/2016/new-dc-os-packages-you-can-use-november-2016/index.html

Would be easier for me grab the content from the PR for next month's post.

**Comment by ryadav88** [comment]
@twk3 Can I also ask you to write a short `getting started` docs for this package so users don't have any issues installing and understanding how it works?

You can create a PR against https://github.com/dcos/examples and refer other examples listed there. These examples will finally go here: https://dcos.io/docs/1.8/usage/tutorials/.

This is also now part of the package submission guidelines: https://github.com/mesosphere/universe#submit-your-package

**Comment by twk3** [comment]
@ssk2 had started a tutorial for it over here: https://github.com/dcos/dcos-website/pull/831

I can cannibalise, and move it to dcos/examples if needed, or maybe @ssk2 can take another look at finishing it off?

Also, came here to say I did a quick test of both CE and EE on dcos 1.8.7, and tested an upgrade scenario.

**Comment by ryadav88** [comment]
@twk3 thanks for letting me know. Looks like thats a blog post. dcos/examples is a more permanent place and acts as an installation guide. These dcos/examples would be listed in the tutorials section here: https://dcos.io/docs/1.8/usage/tutorials/

Yes, you can take the content from the post and add it as an example. But please use the template from the other examples.

File: repo/packages/G/gitlab/3/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "gitlab",
+  "version": "1.0-8.14.1",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://gitlab.com/gitlab-org/gitlab-ce",
+  "maintainer": "support@gitlab.com",
+  "website": "https://about.gitlab.com/",
+  "framework": false,
+  "description": "Collaboration and source control management: code, test, and deploy together!",
```
Comment (ryadav88) [inline repo/packages/G/gitlab/3/package.json]: Add a link to dcos/example at the end of `description`. 
Something like: `Installation Guide: https://github.com/dcos/examples/tree/master/1.8/gitlab`

**Comment by ssk2** [comment]
@twk3, @ryadav88: let's move that tutorial into dcos/examples.

**Comment by ssk2** [comment]
I can take a look at it this afternoon!

**Comment by ryadav88** [comment]
@ssk2 I've added the example based on your blog post.

**Pull request merged**